### DISCUSSION TITLE
SD card refresh

### DISCUSSION THREAD

**Discussion by FinlessBob** [Jyers/Marlin#1232] [General Discussion]
I am about to add an Micro SD to SD adapter like many are doing.  It's on order.  However it appears with this adapter the sd card is not refreshed due to a pin telling the board a sd card was inserted.  Does this firmware have that fix?  If not could this be added per this reddit post?

https://www.reddit.com/r/ender3/comments/i9mmat/refresh_sd_card_with_micro_sd_to_sd_adapter/

Thanks,
Bob

**Comment by Cal-LF** [comment]
@FinlessBob The SD card will refresh with a menu move from Print and back. Also from other main menu moves to the Print menu.

**Reply by FinlessBob** [reply]
Cool!  Thank you for letting me know! :)
